ion/include/ion.h

```diff
@@ -27,6 +27,8 @@ namespace Ion {
 
 void msleep(long ms);
 void usleep(long us);
+extern "C" long millis();
```

Comment:
Would it be possible to use an explicit type?
```suggestion
extern "C" uint32_t millis();
```